index.html

```diff
@@ -4,26 +4,42 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Portfolio</title>
+  <title> Web developers Portfolio  </title>
   <link rel="stylesheet" href="style.css">
+  <link rel="stylesheet" href="script.js">
   <link href="https://fonts.googleapis.com/css2?family=Inter:wght@300&display=swap" rel="stylesheet">
 </head>
 <body>
   
   
     <header class="header">
-      
-      <a href="welcome" class="head">Welcome</a>
-      <nav class="menu-bar">
-        <img src="images/Normal Button/Enabled.png" alt="">
+      <div class="menu-bar">
+        <a href="welcome" class="head">Welcome</a>
+        <nav class="menu-bar2">
+          <ul class="nav-menu">
+            <a href="" onclick="" class="close-button">&times;</a>
+            <li ><a href="#portfolio">Portfolio</a></li>
+            <li><a href="#about">About</a></li>
+            <li><a href="#contact">Contact</a></li>
```

Comment:
On click, the menu user should be moved to the correct part of the screen.

![image](https://user-images.githubusercontent.com/27709832/223805339-1fd89cef-5532-40fc-b096-c6718fcce87b.png)

Currently, this is not the case. Kindly fix it.

![plaze](https://user-images.githubusercontent.com/27709832/223810085-aa859019-90be-4acf-adb9-8659405c2257.gif)